scripts/xcp/logger.pyi

```diff
@@ -0,0 +1,15 @@
+# manually derived from mypy'sy stubgen tool:
```

Comment:
Xapi's build requires xcp-python3-lib https://github.com/xenserver/python-libs/blob/master/xcp/logger.py, could we install it by pip in CI instead of duplicating it for mypy checking?